app/components/UI/WhatsNewModal/whatsNewList.ts

```diff
@@ -7,8 +7,8 @@ import { WhatsNew } from './types';
 export const whatsNew: WhatsNew = {
   // All users that have <6.4.0 and are updating to >=6.4.0 should see
   onlyUpdates: false, // false: Users who updated the app and new installs will see this. true: only users who update will see it
-  maxLastAppVersion: '7.15.0', // Only users who had a previous version <7.15.0 version will see this
-  minAppVersion: '7.15.0', // Only users who updated to a version >= 7.15.0 will see this
+  maxLastAppVersion: '7.19', // Only users who had a previous version <7.19.0 version will see this
+  minAppVersion: '7.14.0', // Only users who updated to a version >= 7.14.0 will see this
```

Comment:
[non-blocker] I think the comments here can be improved. I find the minAppVersion comment to also be misleading since it's not dependent on being an updated version

```suggestion
  maxLastAppVersion: '7.19', // When updating, only users who had a previous version < '7.19' version may see the modal
  minAppVersion: '7.14.0', // Only users whose current version is >= 7.14.0 may see the modal
```

---

for more details about `maxLastAppVersion` and `minAppVersion` please see [related comment in other PR](https://github.com/MetaMask/metamask-mobile/pull/8453#discussion_r1470139639)